features/liquidity/components/StakingSummary.tsx

```diff
@@ -22,16 +18,19 @@ type StakingSummaryProps = {
   maxLiquidity: number
   liquidityAmount: number
   onChangeLiquidity: (liquidityAmount: number) => void
+  maxLiquidityInDollarValue: number
```

Comment:
same max here, also not consistent with above where we use `maxLiquidityDollarValue` without the `in`